logger/format.go

```diff
@@ -0,0 +1,85 @@
+package logger
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+type logEntryStructured struct {
+	Time    time.Time `json:"time"`
```

Comment:
IIRC, the default string serialization format of `time.Time` is not suitable for logging. I'm curious what the output JSON actually looks like here.
You may want to have this struct implement the [json.Marshaler](https://pkg.go.dev/encoding/json#Marshaler) interface so you can format the time explicitly.
Looking at the tests, I don't see any that actually verify the JSON output format.  If you wish to do that and need an example, I have some tests that do that [here](https://github.com/kentik/chf-cmd/blob/master/pkg/queryrelay/relay/http_test.go#L161).

Reply:
Default serialization:
```
{"time":"2022-09-07T19:32:55.049633946Z","name":"./kproxy [2189:versa_br1_our1:404050] [3044075]","level":"info","prefix":"[CHF]","caller":"/home/debian/src/chf/pkg/common/tx/http/http.go:84","message":"HTTP sendTarget 3: online for https://flow.OUR1.kentik.com/chf, UA: Kentik: dirty-babf062ea69594d98297c1de532b1bcf7939c016 Built on Linux 5.10.16.3-microsoft-standard-WSL2 x86_64 [go version go1.18.5 linux/amd64] Debian GNU/Linux 9.13 (stretch) (Tue, 6 Sep 2022 21:31:54 -1000): 2189:versa_br1_our1:404050"}
{"time":"2022-09-07T19:32:56.603166385Z","name":"./kproxy [2189:versa_br2_our1:404055] [3044066]","level":"info","prefix":"[CHF]","caller":"/home/debian/src/chf/pkg/common/tx/http/http.go:84","message":"HTTP sendTarget 3: online for https://flow.OUR1.kentik.com/chf, UA: Kentik: dirty-babf062ea69594d98297c1de532b1bcf7939c016 Built on Linux 5.10.16.3-microsoft-standard-WSL2 x86_64 [go version go1.18.5 linux/amd64] Debian GNU/Linux 9.13 (stretch) (Tue, 6 Sep 2022 21:31:54 -1000): 2189:versa_br2_our1:404055"}
```
Good enough?

Reply:
Yeah actually I think that should be fine.  Thanks.